Detector/DetFCChhTrackerTkLayout/src/TkLayoutEndcap_Geo.cpp

```diff
@@ -16,7 +19,7 @@ static DD4hep::Geometry::Ref_t createTkLayoutTrackerEndcap(DD4hep::Geometry::LCD
   // shorthands
   DD4hep::XML::DetElement xmlDet = static_cast<DD4hep::XML::DetElement>(xmlElement);
   Dimension dimensions(xmlDet.dimensions());
-  double l_overlapMargin = 0.00001;
+  double l_overlapMargin = 0.01;
```

Comment:
To be honest I arrived at this configuration since here both the overlap checks pass and the tracking geometry builds, which wasn't as straightforward as it ought to be. The plan was to move this to TkLayout, so I propose to check the overlaps thoroughly when we move to the next version of the tracker geometry (which is already there, so I would think very soon)?